### ISSUE TITLE
Method couldn't be found in the compiled CloudFormation template

### ISSUE THREAD

**Issue by daniyals** [DianaIonita/serverless-api-gateway-caching#44]
Hello,

Im getting this following error when I try to deploy the serverless code. The app has been in production for quite sometime and now its showing this error and I need to modify the cache settings

> The method ApiGatewayMethodInstant-articleSlugVarGet couldn't be found in the compiled CloudFormation template. Caching settings will not be updated for this endpoint.

```
  instantarticlebyslug:
    handler: services/instantarticles/postbyslug.show
    environment:
      ENDPOINT_URL: <domain>
    events:
      - http:
          path: post/{slug}/instant-articles
          method: get
          caching:
            enabled: true
            cacheKeyParameters:
              - name: request.path.slug
```

I think its the - in instant-articles causing the issue

**Comment by cvr270** [comment]
I ran into a similar issue. As far as I could see the issue is the `-`
`ApiGatewayMethodInstant-articleSlugVarGet` should be translated to `ApiGatewayMethodInstantDasharticleSlugVarGet` in the cloudformation stack, if I'm not mistaken?

**Comment by daniyals** [comment]
Yes I removed the - and it deployed fine.

**Comment by DianaIonita** [comment]
Thanks for finding this issue, I've just released [v1.3.5](https://github.com/DianaIonita/serverless-api-gateway-caching/releases/tag/1.3.5) which should fix it.
